tdrs-backend/tdpservice/settings/cloudgov.py

```diff
@@ -60,7 +61,7 @@ class CloudGov(Common):
     DATABASES = {
         'default': {
             'ENGINE': 'django.db.backends.postgresql',
-            'NAME': database_creds['db_name'],
+            'NAME': f'tdp_db_{cloudgov_space_suffix}_{cloudgov_name}',
```

Comment:
For example, 'tdp_db_dev_sandbox' is one database name.

After looking at this with fresh eyes, I'm not sure what staging or prod would be. For staging, we have "tanf-staging" as the space_name. So I think the db name will be 'tdp_db_staging_staging'.